### ISSUE TITLE
[Suggestion] Bring back the heat bar

### ISSUE THREAD

**Issue by PFSCommissar** [tsunamayo/Starship-EVO#4580]
Although heat death has been removed, being able to monitor the heat is very important for combat, as well as properly engineering a ship to handle the heat loads that it will undergo. Therefore, please bring back the heat bar to the HUD.

**Comment by ProPeach** [comment]
Agreed! Even a smaller percentage heat icon would work great if you don't want it to take up as much GUI space as a bar

**Comment by tsunamayo** [comment]
@PFSCommissar Hi, yes like I will put some sort of an indication when it say > 0.75%, I just dont want to overload the player with info. We might have other status affliction too from weapon, so I need something more low-key. Thanks
